user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -294,7 +296,8 @@ def __build_global_report_summary(self,
                                       all_apps: pd.DataFrame,
                                       total_apps: pd.DataFrame,
                                       unsupported_ops_df: pd.DataFrame,
-                                      output_files_info: JSONPropertiesContainer) -> QualificationSummary:
+                                      output_files_info: JSONPropertiesContainer,
+                                      qual_summary_pd: pd.DataFrame) -> QualificationSummary:
```

Comment:
Is there a way to reduce the diff in that PR? s there a technical reason the `qual_summary_pd` is becoming a new  in all those functions? The way I see it is that  `qual_summary_pd` is part of the other dataframes. 
It will be nice also if we add a comment to describe the new argument. I remember I had difficult time understanding the different between dataframes sent as arguments to functions.

Reply:
removed `qual_summary_pd`, thanks!